cmd/kubeless/ingressList_test.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+	xv1beta1 "k8s.io/client-go/pkg/apis/extensions/v1beta1"
+)
+
+func listIngressOutput(t *testing.T, client kubernetes.Interface, ns, output string) string {
+	var buf bytes.Buffer
+
+	if err := doIngressList(&buf, client, ns, output); err != nil {
+		t.Fatalf("doList returned error: %v", err)
+	}
+
+	return buf.String()
+}
+
+func TestIngressList(t *testing.T) {
+	ing1 := xv1beta1.Ingress{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "foo",
+			Namespace: "myns",
+		},
+		Spec: xv1beta1.IngressSpec{
+			Rules: []xv1beta1.IngressRule{
+				{
+					Host: fmt.Sprintf("%s.%s", "foo", "example.com"),
```

Comment:
Ehh... it's just for making clear the format of <function_name>.<domain> Yes it can be the literal `foo.example.com` but I thought it is easier to follow in further development when somebody looks at the unit test and get the idea of Host.